frontend/src/pages/Login.jsx

```diff
@@ -1,8 +1,27 @@
 import * as React from "react";
+import { useState } from "react";
 import "./Login.css";
 import TextField from "@mui/material/TextField";
 import Button from "@mui/material/Button";
 const App = () => {
+    const [email, setEmail] = useState("")
+    const [password, setPassword] = useState("")
+    const onSubmit = async (e) => {
+        console.log(email + " " + password)
+        let result = await logIn(email, password)
+    }
+    const logIn = async (email, password) => {
+        try {
+            let result = await axios.get(
+                "http://localhost:8080/api/user_authenticate?username=" + email + "&password=" + password
```

Comment:
The backend api call for login would be: ".../api/login?username=&password="

Reply:
fixed